R/approach.R

```diff
@@ -43,31 +39,13 @@ setup_approach.combined <- function(internal, ...) {
 
 #' Generate data used for predictions and Monte Carlo integration
 #'
-#' @note
-#' There is currently a warning when calling `roxygen2::roxygenise()`.
-#' Warning message:
-#' inheritParams failed in topic "prepare_data", "prepare_data.categorical", "prepare_data.copula",
-#' "prepare_data.ctree", "prepare_data.empirical", "prepare_data.gaussian",
-#' "prepare_data.independence", "prepare_data.timeseries", and "prepare_data.vaeac".
-#' ✖ All parameters are already documented; none remain to be inherited.
-#'
-#' This occurred after Lars updated the code as he now documents all parameters here. This was not
-#' the case before, as the roxygen had not been updated. Can solve this by removing, e.g.,
-#' `param ... Currently not used.` below, or remove rdname from the subfunctions and rather
-#  use inherateparams.
-#'
-#'
-#' @param internal List. Holds all parameters, data, functions and computed objects used
-#' within \code{\link[=explain]{explain()}}. The list contains one or more of the elements
-#' `parameters`, `data`, `objects`, `output`.
-#' @param index_features Positive integer vector. Specifies the indices of combinations to
-#' apply to the present method. `NULL` means all combinations. Only used internally.
-#'
```

Comment:
As mentioned earlier, we use inheritParams to avoid copying. I inherited from default_doc_explain (rather than default_doc) as the documentation of explain is affected by this doc, and for that the explain function, it can be confusing if one is told to pass the internal object (hence "not used" although it is not strictly correct for the prepare_data function).

Reply:
Thank you for clarifying this, and it makes sense. I will try to remember this going forward.